src/Components/SkillPageRow/SkillRow.test.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import SkillRow from './SkillRow';
+
+//Test to see if the container for the skill page components is rendered
+describe('SkillsRow.js', () => {test("User should see a container for the components of the skill page", () => {
```

Comment:
The describe() method is used to group similar tests together; I would not suggest using describe() per test as the test description does that itself.
https://jestjs.io/docs/api#describename-fn